src/frontend/screens/Settings/sections/GamesSettings/index.tsx

```diff
@@ -42,6 +42,7 @@ import SyncSaves from '../SyncSaves'
 import FooterInfo from '../FooterInfo'
 import { Tabs, Tab } from '@mui/material'
 import { GameInfo } from 'common/types'
+import EnableMsync from '../../components/EnableMsync'
```

Comment:
This import could be inside the above one that is already importing other components from the same folder

Reply:
fixed!